### DISCUSSION TITLE
Static Media Failure after Upgrade

### DISCUSSION THREAD

**Discussion by darrenwoods3** [netbox-community/netbox#10042] [Q&A]
After performing a successfull upgrade and restarting the netbox and apache services, i receive this message.  Help would be greatly appreciated.
<img width="826" alt="Screen Shot 2022-08-16 at 11 10 44 PM" src="https://user-images.githubusercontent.com/109082632/185033909-9c064dd8-6746-48ce-b219-2d45b27f0514.png">

**Comment by promasu** [comment]
Hi!

You are sure that all permissions are correct and that the static file dir inside your Webserver configuration matches the folder netbox writes its static files to?

**Comment by darrenwoods3** [comment]
I don’t know if the permissions are correct and I don’t know where to look.  All I did was an upgrade to the latest version.  Any help will be appreciated.

Thanks,
Darren
